DigitalLearningSolutions.Web/ViewModels/LearningMenu/CourseCompletionViewModel.cs

```diff
@@ -33,7 +33,7 @@ public CourseCompletionViewModel(IConfiguration config, CourseCompletion courseC
             PostLearningPasses = courseCompletion.PostLearningPasses;
             SectionCount = courseCompletion.SectionCount;
 
-            PercentageTutorialsCompleted = Convert.ToInt32(Math.Round(courseCompletion.PercentageTutorialsCompleted));
+            PercentageTutorialsCompleted = Convert.ToInt32(Math.Floor(courseCompletion.PercentageTutorialsCompleted));

```

Comment:
I think we're doing this a few other places as well (rounding when we should be flooring). I'll raise a bug 🙂

Reply:
Yeah I agree I've definitely written rounding code elsewhere 🙂

Reply:
HEEDLS-275 raised 🙂 I've assigned it to @SChandarana as he has less work but feel free to discuss it together of course 🙂